docs/src/integrate/integration-tutorial.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Integrate with the Node.js API Tutorial
+eleventyNavigation:
+    key: integrate with the node.js api tutorial
+    parent: integrate eslint
+    title: Integrate with the Node.js API Tutorial
+    order: 1
+---
+
+This guide walks you through integrating the `ESLint` class to lint files and retrieve results, which can be useful for creating integrations with other projects.
+
+## Why Create an Integration?
+
+You might want to create an ESLint integration if you're creating developer tooling, such as the following:
+
+* **Code editors and IDEs**: Integrating ESLint with code editors and IDEs can provide real-time feedback on code quality and automatically highlight potential issues as you type. Many editors already have ESLint plugins available, but you may need to create a custom integration if the existing plugins do not meet your specific requirements.
+
+* **Custom linter tools**: If you're building a custom linter tool that combines multiple linters or adds specific functionality, you may want to integrate ESLint into your tool to provide JavaScript linting capabilities.
+
+* **Code review tools**: Integrating ESLint with code review tools can help automate the process of identifying potential issues in the codebase.
+
+* **Learning platforms**: If you are developing a learning platform or coding tutorial, integrating ESLint can provide real-time feedback to users as they learn JavaScript, helping them improve their coding skills and learn best practices.
+
+* **Developer tools** If you're creating developer tools, like a testing framework or a bundler, you may want to integrate ESLint to provide linting capabilities.
+
+## What You'll Build
+
+In this guide, you'll create a simple Node.js project that uses the `ESLint` class to lint files and retrieve results.
+
+## Requirements
+
+This tutorial assumes you are familiar with JavaScript and Node.js.
+
+To follow this tutorial, you'll need to have the following:
+
+* Node.js (v12.0.0 or higher)
+* npm or Yarn package manager
+* A text editor
+
+## Step 1: Setup
+
+First, create a new project directory:
+
+```shell
+mkdir eslint-integration
+cd eslint-integration
+```
+
+Initialize the project with a `package.json` file:
+
+```shell
+npm init -y
+```
+
+Install the `eslint` package as a dependency:
```

Comment:
It may be worth calling out that the package is installed as a dependency and not a dev dependency for this use case.

Reply:
updated